docs/tutorials/csharp/stable-diffusion-csharp.md

```diff
@@ -0,0 +1,331 @@
+---
+title: Stable Diffusion with C#
+description: In this tutorial we will learn how to do inferencing for the popular Stable Diffusion deep learning model in C#.
+parent: Inference with C#
+grand_parent: Tutorials
+has_children: false
+nav_order: 1
+---
+
+# Inference Stable Diffusion with C# and ONNX Runtime
+{: .no_toc }
+
+In this tutorial we will learn how to do inferencing for the popular Stable Diffusion deep learning model in C#. Stable Diffusion models denoise a static image to create an image that represents the text prompt given by the user.
+
+```text
+"make a picture of green tree with flowers around it and a red sky" 
+```
+
+|----------------------|----------------------|
+| <img src="../../../images/latents-noise-example.png" width="256" height="256" alt="Image of browser inferencing on sample images."/> | <img src="../../../images/sample-output-stablediff.png" width="256" height="256" alt="Image of browser inferencing on sample images."/> |
+
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Prerequisites
```

Comment:
Do you have to build Onnxruntime Extensions from source to use this?

Reply:
No. I included the dll in the project

Reply:
I tried to run it and it could not find it

Reply:
Does it need to be included in the csproj?